libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/lifecycle/UICoordinator.kt

```diff
@@ -21,33 +22,49 @@ import kotlinx.coroutines.launch
  */
 @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
 abstract class UICoordinator<T : ViewGroup>(
-    private val viewGroup: T
+    protected val viewGroup: T
 ) : MapboxNavigationObserver {
 
     lateinit var coroutineScope: CoroutineScope
 
+    protected var viewBinder: Binder<T>? = null
+    protected var attachedObserver: MapboxNavigationObserver? = null
+
     @CallSuper
     override fun onAttached(mapboxNavigation: MapboxNavigation) {
-        var attachedObserver: MapboxNavigationObserver? = null
         coroutineScope = MainScope()
 
         coroutineScope.launch {
-            mapboxNavigation.flowViewBinders().collect { viewBinder ->
+            mapboxNavigation.flowViewBinders().collect { binder ->
+                logD(
+                    this.javaClass.simpleName,
+                    "binder: $viewBinder -> $binder"
+                )
                 attachedObserver?.onDetached(mapboxNavigation)
-                attachedObserver = viewBinder.bind(viewGroup)
+                viewBinder?.also { unbind(it, viewGroup) }
```

Comment:
Discussed offline, linking an issue this is resolving https://github.com/mapbox/mapbox-navigation-android/issues/5546

Reply:
It looks like trying to solve the issue with MapView not loading upon backgrounding/foregrounding is irrelevant to this ticket, if that is what we are trying to solve here. Can we rather identify these issue and cut tickets so that it could be handled separately thereby making the PR smaller. In other words if `unbind` is added to solve the [problem](https://github.com/mapbox/navigation-sdks/issues/1506) is it possible to not introduce it here and hande it separately?

Reply:
I fixed this issue as one of the first things when I started working on InfoPanel ticket. I am not going to rip this out just to re-add it again in a separate PR.
I'll refrain from fix anything that is outside active ticket scope in the future.